app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return phrase.lower().count(letter.lower())
```

Comment:
The function `count_occurrences` should return an integer representing the number of occurrences of `letter` in `phrase`. However, the `count` method is correctly used, but the result is not converted to an integer. The division operator `/` is not needed here, as `count` already returns an integer. Ensure that the function returns an integer directly by removing the division operator.

Reply:
where do you see the sign "/" ?